accelerator/common_core.c

```diff
@@ -450,3 +453,40 @@ status_t ta_send_bundle(const iota_config_t* const tangle, const iota_client_ser
 
   return SC_OK;
 }
+
+status_t ta_get_bundle_by_addr(const iota_client_service_t* const service, flex_trit_t const* const addr,
```

Comment:
Is it necessary to call `find_transaction` API multiple times? I was thinking just call it and provide address. We could just refactor `ta_get_bundle` in this way.

Reply:
I think refactor `ta_get_bundle` is a great suggestion, but if possible, I want to do that in another PR.  I am evaluating about which implementation of get_bundle is better, entangled of ours.